### PR TITLE
tiongMax: add profile photo

### PR DESCRIPTION
**Pull request by tiongMax**
Ok i accidentally created another branch to add the picture, can merge the other one if there is no changes.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Tiong Meng Sheng
 
 <img src="images/johndoe.png" width="200px">
```
Comment (brein62) [inline docs/AboutUs.md:55]: u forgot to update the image link lol

**Review by brein62 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (tiongMax) [reply docs/AboutUs.md:55]: oh ya ahahaha

**Review by brein62 — APPROVED** [review]
LGTM!

File: docs/images/tiongMax.png
Comment (jordankanghm) [inline docs/images/tiongMax.png:1]: the rizzler😩😩

**Review by jordankanghm — APPROVED** [review]

**Pull request merged**